<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="0" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-08-12T18:04:59 IST" name="Default suite" finished-at="2024-08-12T18:05:29 IST" duration-ms="30319">
    <groups>
    </groups>
    <test started-at="2024-08-12T18:04:59 IST" name="Default test" finished-at="2024-08-12T18:05:29 IST" duration-ms="30319">
      <class name="testscripts.Login">
        <test-method is-config="true" signature="setBrowser()[pri:0, instance:testscripts.Login@351d00c0]" started-at="2024-08-12T18:04:59 IST" name="setBrowser" finished-at="2024-08-12T18:05:00 IST" duration-ms="1606" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setBrowser -->
        <test-method signature="userLoginWithInvalidCredentials(java.lang.String,java.lang.String)[pri:0, instance:testscripts.Login@351d00c0]" started-at="2024-08-12T18:05:00 IST" name="userLoginWithInvalidCredentials" data-provider="invalidUserCredentials" finished-at="2024-08-12T18:05:08 IST" duration-ms="7630" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[12345678678]]>
              </value>
            </param>
          </params>
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA["C:\Users\pooja\git\Seleniumbasics\Seleniumbasics\src\test\resources\TestData1.xlsx" (The filename, directory name, or volume label syntax is incorrect)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: "C:\Users\pooja\git\Seleniumbasics\Seleniumbasics\src\test\resources\TestData1.xlsx" (The filename, directory name, or volume label syntax is incorrect)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:158)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:112)
at utilities.ExcelUtility.getStringData(ExcelUtility.java:19)
at testscripts.Login.userLoginWithInvalidCredentials(Login.java:34)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userLoginWithInvalidCredentials -->
        <test-method is-config="true" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:testscripts.Login@351d00c0]" started-at="2024-08-12T18:05:08 IST" name="closeBrowser" finished-at="2024-08-12T18:05:08 IST" duration-ms="258" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=userLoginWithInvalidCredentials status=FAILURE method=Login.userLoginWithInvalidCredentials(java.lang.String,java.lang.String)[pri:0, instance:testscripts.Login@351d00c0] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="setBrowser()[pri:0, instance:testscripts.Login@351d00c0]" started-at="2024-08-12T18:05:08 IST" name="setBrowser" finished-at="2024-08-12T18:05:09 IST" duration-ms="1158" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setBrowser -->
        <test-method signature="userLoginWithInvalidCredentials(java.lang.String,java.lang.String)[pri:0, instance:testscripts.Login@351d00c0]" started-at="2024-08-12T18:05:09 IST" name="userLoginWithInvalidCredentials" data-provider="invalidUserCredentials" finished-at="2024-08-12T18:05:21 IST" duration-ms="12053" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[123456]]>
              </value>
            </param>
          </params>
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA["C:\Users\pooja\git\Seleniumbasics\Seleniumbasics\src\test\resources\TestData1.xlsx" (The filename, directory name, or volume label syntax is incorrect)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: "C:\Users\pooja\git\Seleniumbasics\Seleniumbasics\src\test\resources\TestData1.xlsx" (The filename, directory name, or volume label syntax is incorrect)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:158)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:112)
at utilities.ExcelUtility.getStringData(ExcelUtility.java:19)
at testscripts.Login.userLoginWithInvalidCredentials(Login.java:34)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userLoginWithInvalidCredentials -->
        <test-method is-config="true" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:testscripts.Login@351d00c0]" started-at="2024-08-12T18:05:21 IST" name="closeBrowser" finished-at="2024-08-12T18:05:22 IST" duration-ms="202" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=userLoginWithInvalidCredentials status=FAILURE method=Login.userLoginWithInvalidCredentials(java.lang.String,java.lang.String)[pri:0, instance:testscripts.Login@351d00c0] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method is-config="true" signature="setBrowser()[pri:0, instance:testscripts.Login@351d00c0]" started-at="2024-08-12T18:05:22 IST" name="setBrowser" finished-at="2024-08-12T18:05:23 IST" duration-ms="1097" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setBrowser -->
        <test-method signature="userLoginWithInvalidCredentials(java.lang.String,java.lang.String)[pri:0, instance:testscripts.Login@351d00c0]" started-at="2024-08-12T18:05:23 IST" name="userLoginWithInvalidCredentials" data-provider="invalidUserCredentials" finished-at="2024-08-12T18:05:29 IST" duration-ms="6007" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin12]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[34645575]]>
              </value>
            </param>
          </params>
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA["C:\Users\pooja\git\Seleniumbasics\Seleniumbasics\src\test\resources\TestData1.xlsx" (The filename, directory name, or volume label syntax is incorrect)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: "C:\Users\pooja\git\Seleniumbasics\Seleniumbasics\src\test\resources\TestData1.xlsx" (The filename, directory name, or volume label syntax is incorrect)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:158)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:112)
at utilities.ExcelUtility.getStringData(ExcelUtility.java:19)
at testscripts.Login.userLoginWithInvalidCredentials(Login.java:34)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- userLoginWithInvalidCredentials -->
        <test-method is-config="true" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:testscripts.Login@351d00c0]" started-at="2024-08-12T18:05:29 IST" name="closeBrowser" finished-at="2024-08-12T18:05:29 IST" duration-ms="236" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=userLoginWithInvalidCredentials status=FAILURE method=Login.userLoginWithInvalidCredentials(java.lang.String,java.lang.String)[pri:0, instance:testscripts.Login@351d00c0] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- testscripts.Login -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
